docs/assets/css/custom.css

```diff
@@ -0,0 +1,23 @@
+.logo-config {
+  vertical-align: middle !important;
+  width: 100%;
+  height: auto;
+  width: calc(100px + 6 * ((100vw - 320px) / 680))
+}
+
+.c-footer-style {
+  padding: 20px;
+  background: #333333;
+  color: #ffffff;
+}
+.footer-heading-row {
+  font-size: 21px !important;
+  font-weight: bold;
+  margin-bottom: calc(40px + (48 - 40) * ((100vw - 360px) / (1600 - 360))) !important;
```

Comment:
I'm not exactly sure what's going on here, but I think it's probably simpler to just give the `li` item an `mb-4` and then remove this line. What do you think?

Reply:
Yeah, this worked. I thing Bootstrap works like magic, I think as I work more I will stick with the default configuration provided by framework. Thanks!